Sage/carbon
# refactor(portrait): convert class to functional component (FE-3659)

Fixes FE-3659

### Proposed behaviour

Component should behave exactly as before when used. 

### Checklist




#### QA

- [ ] Tested in sandbox/storybook

### Testing instructions

Use the storybook docs to confirm that the component still works. Unit tests have been run. A Cypress test already exists but might need to be added to.
